src/coreclr/vm/dllimportcallback.cpp

```diff
@@ -229,18 +256,36 @@ PCODE TheUMEntryPrestubWorker(UMEntryThunkData * pUMEntryThunkData)
     if (pUMEntryThunkData->IsCollectedDelegate())
         CallbackOnCollectedDelegate(pUMEntryThunkData);
 
-    INSTALL_MANAGED_EXCEPTION_DISPATCHER;
-    // this method is called by stubs which are called by managed code,
-    // so we need an unwind and continue handler so that our internal
-    // exceptions don't leak out into managed code.
-    INSTALL_UNWIND_AND_CONTINUE_HANDLER;
+    PCODE callTarget = (PCODE)0;
```

Comment:
None of the change in this method should be needed. We can return ` (PCODE)pUMEntryThunkData->GetCode();` in all cases. For the interpreter case, it is going to immediately call into TheUMEntryPrestubWorker again, set thread static, and return.